.changeset/wicked-berries-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+App's "about" section will be rendered as a plain text, instead of a markdown
```

Comment:
**issue**: do you need two changeset files?

Reply:
No, but IMO its better.  These changes are not related. So you write a change, add a changeset and commit. Later you add another change and do the same.

This gives us clear timeline. 

In generated changeset it will be 2 paragraphs (because they are not related, they just happened to be on the same PR). They will be linked to separate commits too

See example changeset of one of the apps
https://github.com/saleor/apps/releases/tag/saleor-app-data-importer%401.8.0